### ISSUE TITLE
在--problem=translate_enzh_sub50k加上_rev进行中译英训练时，模型表现非常差

### ISSUE THREAD

**Issue by gaolinjie** [yanwii/machine-translation#7]
INFO:tensorflow:Saving dict for global step 300000: global_step = 300000, loss = 5.3842344, metrics-translate_enzh_sub50k_rev/targets/accuracy = 0.2301816, metrics-translate_enzh_sub50k_rev/targets/accuracy_per_sequence = 0.000856898, metrics-translate_enzh_sub50k_rev/targets/accuracy_top5 = 0.41271442, metrics-translate_enzh_sub50k_rev/targets/approx_bleu_score = 0.01490011, metrics-translate_enzh_sub50k_rev/targets/neg_log_perplexity = -5.3780227, metrics-translate_enzh_sub50k_rev/targets/rouge_2_fscore = 0.055390585, metrics-translate_enzh_sub50k_rev/targets/rouge_L_fscore = 0.21191081

请问下能这样直接加_rev 进行中译英训练吗？还是需要在problem.py中修改什么？谢谢！

**Comment by yanwii** [comment]
最好不要，没有去做这个实现，可能会有问题。
